packages/liquidator/index.js

```diff
@@ -190,49 +223,47 @@ async function run({
       collateralDecimals: Number(collateralDecimals),
       syntheticDecimals: Number(syntheticDecimals),
       priceFeedDecimals: Number(priceFeed.getPriceFeedDecimals()),
-      priceFeedConfig
+      priceFeedConfig,
+      liquidatorConfig
     });
 
     // The EMP requires approval to transfer the liquidator's collateral and synthetic tokens in order to liquidate
     // a position. We'll set this once to the max value and top up whenever the bot's allowance drops below MAX_INT / 2.
-    // If the contract is expired, the liquidator can only withdraw disputes so there is no need to set allowances.
-    if (!IS_EXPIRED) {
```

Comment:
This isn't a problem IMO to remove the `IS_EXPIRED` check, as it prevents sending allowance transactions in only a strange edge case: where the bot is running on an expired EMP that it has not approved yet.

Reply:
ye, that's why I removed it. it was unnecessary logic that would really never apply in reality.